lib/datasource/github-tags/index.ts

```diff
@@ -166,3 +157,34 @@ export async function getReleases({
   );
   return dependency;
 }
+
+function normalizeVersion(version: string): string {
+  return version?.replace(/^v/, '');
```

Comment:
What about using the `semver` package's functions like this: `semver.valid(semver.coerce('v2')) // '2.0.0'` ?

Reply:
I didn't want to rely on particular version schemes in these datasources. `semver.coerce` should return wide range of normalized result strings seldom returning null, but `/^v/` stripping always returns string.

Reply:
Actually I think GitHub releases raw result includes the exact tag, which would be a better way to match?